src/test/java/com/aws/greengrass/clientdevices/auth/CertificateManagerTest.java

```diff
@@ -160,18 +146,13 @@ void Given_defaultCAConfiguration_THEN_returnsAutoGeneratedCA() throws Exception
         X509Certificate caCertificate = CertificateHelper.createCACertificate(
                 keyPair, Date.from(now), Date.from(now.plusSeconds(10)), "Custom Core CA");
 
-        URI privateKeyUri = new URI("file:///private.key");
-        URI certificateUri = new URI("file:///certificate.pem");
-
-        // TODO: Write the actual certificate to the file system and avoid mocking the security service. Doing
-        //  this is a bad given we are exposing implementation details on the test.
-        when(securityServiceMock.getKeyPair(privateKeyUri, certificateUri)).thenReturn(keyPair);
```

Comment:
No more mocking we are writing them and letting the things actually execute the ported methods. This gives us confidence they are working